notebookPlots.jmd

```diff
@@ -18,137 +13,166 @@ effects = getODEparams(ps, conc_l)
 g0 = g1[1] + g2[1]
 T = 170
 ```
+
+### Lapatinib
+
 #### Plotting the gradient
+
```

Comment:
It looks like you might need to have some non-header text here to ensure the figure is left-justified.